docs/provisioning.md

```diff
@@ -0,0 +1,111 @@
+# Provisioning the IDR
+
+## Overview of the architecture
+
+The IDR contains two main groups of servers:
+
+The production (public-facing) IDR (3 servers):
+- Database
+- OMERO.server
+- Nginx gateway
+
+The virtual analysis environment (VAE) IDR (3 servers):
+- Database
+- OMERO.server
+- Docker Manager
+
+
+## Ansible prerequisites
+
+Almost all of the provisioning and deployment in the IDR is done using Ansible 2.1.
+All ansible commands should be run from a shell in the [`ansible`](../ansible) directory.
+
+You must first install the required galaxy roles:
+
+    ansible-galaxy install -r requirements.yml
+
+The [`ansible.cfg`](../ansible/ansible.cfg) configuration file will install the roles into the [`vendor`](../ansible/vendor) directory.
+
+
+## OpenStack
+
+The IDR is currently hosted on OpenStack, and we have an [example Ansible playbook](../ansible/openstack-create-infrastructure.yml) for provisioning compute, storage and networking.
+The Ansible openstack modules require the `shade` python module.
+
+This playbook will create two networks `idr` and `idr-a` for the production and analysis servers, and multiple instances and storage volumes.
```

Comment:
- https://trello.com/c/TUdosdRT/235-de-couple-idr-production-and-analysis-openstack-playbooks
- https://trello.com/c/3SPDElcJ/169-jupyterhub-on-docker-swarm